container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -3066,6 +3067,12 @@ configuration:
     # Really a loggregator_agent job property
     properties.zone: '"((KUBE_AZ))"'
 variables:
+- name: ACCEPTANCE_TEST_FLAKE_ATTEMPTS
+  options:
+    default: 3
+    description: >
+      The number of times Ginkgo will run a test before treating it as a failure.
+      Individual failed runs will still be reported in the test output.
```

Comment:
I know it's implied in the name, but we should mention in the description that this is for CATS (unless you want this to be for other tests too, in which case say that).

Reply:
I changes it to `...run a CATS test...`

I also added support for `-flakeAttempts` to SITS, but it is using a separate config variable.